tests/test_helpers.py

```diff
@@ -29,14 +29,15 @@ def test_initials_returns_first_and_middle():
 
 
 @freeze_time("2023-08-18")
-def test_sns_log(caplog, stubbed_sns_client):
+def test_sns_log_returns_start_message(stubbed_sns_client):
```

Comment:
Isn't this test doing more than this now? I'd make it more generic

Reply:
Fixed the name! Thank you for catching this.